PYME/Acquire/HTTPSpooler.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, filename, frameSource, frameShape, **kwargs):
         self.clusterFilter = kwargs.get('serverfilter', CLUSTERID)
         self._buffer = []
         
-        self.buflen = 50
+        self.buflen = config.get('http-chunk-size', 50)
```

Comment:
```suggestion
        self.buflen = config.get('httpspooler-chunksize', 50)
```

Make it specific to the spooler. 

Also collapsed chunksize to follow implicit config convention `<module/program>-<setting>` of using a hyphen as a separator between the program and the setting (rather than a generic word separator) - TODO document this.